### ISSUE TITLE
"target entity" disregards visibility / occlusion

### ISSUE THREAD

**Issue by redbrain** [SkriptLang/Skript#3675]
### Description
Target entity returns the entity in a player's crosshair, regardless of whether the entity is behind a wall, or otherwise not visible.
I don't know if this is expected behavior or not.

### Steps to Reproduce

This skript was loaded:
```
on left click with stick:
    if target entity of player is set:
        if target entity is player:
            message "target entity is player"
        else:
            message "target entity isn't player"
    else:
        message "there is no target entity"
```
### Expected Behavior
When clicking a wall that an entity is directly behind, I expected the output "there is no target entity".
However, I get "target entity isn't player" (since it was a Husk)

### Errors / Screenshots
Immediately after click (crosshair in that position):
![image](https://user-images.githubusercontent.com/42494361/103488525-4a05c200-4ddb-11eb-8a0b-9a19c8f99a75.png)
Position of entity:
![image](https://user-images.githubusercontent.com/42494361/103488541-6f92cb80-4ddb-11eb-9ae0-44bc4cea5e6d.png)

### Server Information
Paper 1.16.4, Skript 2.5.3 (console says that this is the latest stable version)
Also running the SkDragon addon, although I doubt this affects the behavior.

**Comment by ShaneBeee** [comment]
I noticed this the other day too.

Skript uses an odd method to get the target entity, im assuming cause it was written long before there was a Bukkit API for target entity. 

This is an easy fix, I will tackle at some point soon.

**Comment by sovdeeth** [comment]
Fixed by #5947
